test/game_teams_collection_test.rb

```diff
@@ -7,25 +7,40 @@
 class GameTeamsCollectionTest < Minitest::Test
 
   def setup
-    @locations = './data/little_game_teams.csv'
-    @gtc = GameTeamsCollection.new(@locations)
+    @game_team_data = CSV.read('./data/little_game_teams.csv',
+                              headers: true,
+                              header_converters: :symbol)
+    @gtc = GameTeamsCollection.new(@game_team_data)
   end
 
   def test_it_exists
     assert_instance_of GameTeamsCollection, @gtc
   end
 
-  def test_it_can_create_an_array_of_game_data_objects
+  def test_it_can_create_an_array_of_game_team_objects
     assert_instance_of Array, @gtc.game_teams
     assert_instance_of GameTeams, @gtc.game_teams.first
   end
 
   def test_it_has_as_many_teams_as_data_lines
-    assert @gtc.game_teams.length == CSV.readlines(@locations, headers: true).size
+    assert_equal @gtc.game_teams.size, @game_team_data.size
   end
 
   def test_it_has_real_data
-    assert_equal "LOSS", @gtc.game_teams.first.result
+    assert_equal 44.8, @gtc.game_teams.first.face_off_win_percentage
```

Comment:
While I did create fixture files, I left the entry the same, so all these tests should still pass!